sdcm/tester.py

```diff
@@ -982,6 +1003,9 @@ def setUp(self):  # noqa: PLR0912, PLR0915
         if (not self.params.get("cluster_backend").startswith("k8s") and
                 any([self.params.get('client_encrypt'), self.params.get('server_encrypt')])):
             create_ca(self.localhost)
+            if self.params.get('client_encrypt'):
+                cqlshrc_file = get_data_dir_path('ssl_conf', 'client', 'cqlshrc')
+                update_cqlshrc(cqlshrc_file, client_encrypt=self.params.get('client_encrypt'))
```

Comment:
if we do it only under this if, it's kind of weird to pass in `client_encrypt` as parameter 

i.e. there no case we'll call it with `false` now, so we can rename the function and remove the argument